.github/ISSUE_TEMPLATE/issueform.yaml

```diff
@@ -0,0 +1,42 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, triage]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
```

Comment:
I am not sure about this one. If we need more details regarding the issue, we can still add a comment to the issue tagging the creator. What do you think?

Reply:
Yeah I do agree. I will take it away.